docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
-# Confidence Intervals
-By navigating to the detail view of any experiment, you can view the effect that any of the treatments have had on each of the selected metrics for that experiment. You can get to the detail view of any experiment by clicking the **Experiments** tab on the left panel and then clicking on the experiment you're interested in from the list. The effect on each metric comes with an accompanying confidence interval. This confidence interval tells you the upper and lower bound of the effect on the metric in addition to the point estimate of the effect. We show confidence intervals for the ratio between treatment and control, rather than their absolute difference.
+# Lift Estimates and Confidence Intervals
```

Comment:
I wonder if it's useful to add a short intro paragraph with TLDR and content in this page as it's pretty long